### ISSUE TITLE
Clicking on friend name 

### ISSUE THREAD

**Issue by hirohtk** [hirohtk/weather#28]
Should bring up (?) a card next to currentWeather showing your friend's weather status.  

possibly handle this by having each user update their location/coordinates every time they log in?

**Comment by scotthiraki** [comment]
> Should bring up (?) a card next to currentWeather showing your friend's weather status.
> 
> possibly handle this by having each user update their location/coordinates every time they log in?

yes, that sounds good, or it could be a popup when they mouse over their friends name. and then by clicking on the popup, it can show more information
